PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,104 +1,59 @@
-<!-- Please include here ALL the steps necessary to test your Pull Request. If
-it includes tests (and most should), outline here the steps needed to run the
-tests. If not, lay out the manual testing procedure and please explain why
-tests are unnecessary for this Pull Request. -->
+<!-- Please include here ALL the steps necessary to test your Pull Request.
+If your PR includes tests (and most should), provide the steps needed to run the tests.
+If not, please provide step-by-step instructions to test the PR manually and explain why your PR does not need tests. -->
 
-## If this is a bug fix, what versions of Traffic Control are affected?
-<!-- If this PR fixes a bug, please list here all of the affected versions - to
-the best of your knowledge. It's also pretty helpful to include a commit hash
-of where 'master' is at the time this PR is opened (if it affects master),
-because what 'master' means will change over time. For example, if this PR
-fixes a bug that's present in master (at commit hash '1df853c8'), in v4.0.0,
-and in the current 4.0.1 Release candidate (e.g. RC1), then this list would
-look like:
 
-- master (1df853c8)
-- 4.0.0
-- 4.0.1 (RC1)
-
-If you don't know what other versions might have this bug, AND don't know how
-to find the commit hash of 'master', then feel free to leave this section
-blank (or, preferably, delete it entirely).
+## If this is a bugfix, which Traffic Control versions contained the bug?
+<!-- Delete this section if the PR is not a bugfix, or if the bug is only in the master branch.
+Examples:
+- 5.1.2
+- 5.1.3 (RC1)
  -->
 
 
 ## The following criteria are ALL met by this PR
-<!-- Check the boxes to signify that the associated statement is true. To
-"check a box", replace the space inside of the square brackets with an 'x'.
-e.g.
-
-- [ x] <- Wrong
-- [x ] <- Wrong
-- [] <- Wrong
-- [*] <- Wrong
-- [x] <- Correct!
-
--->
-
-- [ ] This PR includes tests OR I have explained why tests are unnecessary
-- [ ] This PR includes documentation OR I have explained why documentation is unnecessary
-- [ ] This PR includes an update to CHANGELOG.md OR such an update is not necessary
-- [ ] This PR includes any and all required license headers
-- [ ] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)
-
-
-## Additional Information
-<!-- If you would like to include any additional information on the PR for
-potential reviewers please put it here.
-
-Some examples of this would be:
-
-- Before and after screenshots/gifs of the Traffic Portal if it is affected
-- Links to other dependent Pull Requests
-- References to relevant context (e.g. new/updates to dependent libraries,
-mailing list records, blueprints)
-
-Feel free to leave this section blank (or, preferably, delete it entirely).
--->
+<!-- Check each box to signify that the associated statement is true. For example:
```

Comment:
We should have these checklists checked by default IMO and maybe reword the section title:
```
## PR submission checklist
[x] This PR has tests <!-- If not, please delete this text and explain why this PR does not need tests. -->
[x] This PR has documentation <!-- If not, please delete this text and explain why this PR does not need documentation. -->
[x] This PR includes a CHANGELOG.md entry <!-- A fix for a bug from an ATC release, an improvement, or a new feature should include a changelog entry. -->
[x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)
```
That way, PRs that meet the criteria don't have to change anything in the checklist at all or check any boxes. We also don't have to tell people how to properly check a box.

Reply:
Checkboxes checked in c8885b4d9e

Reply:
Thoughts on removing checkbox instructions?

Reply:
At the very least, we should be able to remove the section-wide instructions (removed in 8a2f28cfd3). Do you think the inline instructions should be removed, also?

Reply:
> We should have these checklists checked by default IMO and maybe reword the section title

Committed rewordings in 844e350032

Reply:
No, I think the inline instructions are fine, I was just talking about the "how do I check a checkbox" instructions.